glacier/src/actions/add-field.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldAction = ReduxStandardAction<"ADD_FIELD", Field[]>;
+
+export function createAddFieldAction(fields: Field[]): AddFieldAction {
+    return {type: "ADD_FIELD", payload: fields};
```

Comment:
How else would you ensure only one instance of a specific data soure handles the field addition?